### ISSUE TITLE
bad request: missing Authorization header

### ISSUE THREAD

**Issue by 3xarkun** [microsoft/vscode-copilot-release#879]
Copilot does not work anymore. I noticed in output info that the request is sending this response : 
bad request: missing Authorization header

I tried logout and login again but it still doesn't work.

**Comment by mikeltsan** [comment]
Same issue here. Auto compelete is not working. Copilot chat is working for me though, but not code completion.

**Comment by fwsGonzo** [comment]
It's not working here as well.  No code completion

**Comment by goodallman** [comment]
Same here. Weird thing is, that logs are not showing anything "bad". Chat is working with no problems, code completions are just stuck and loading like 10 seconds before the time out.

**Comment by Fdawgs** [comment]
Also hitting it here.

Output from VS Code:
![image](https://github.com/microsoft/vscode-copilot-release/assets/43814140/433e3924-3292-4aa1-b5e4-5aedef10fa85)

**Comment by 3xarkun** [comment]
Yeah, i checked a lot of things, made diagnostics, installed a previous version of vscode, nothing seems to work.

**Comment by kilianpelissier** [comment]
Same for me, autocompelete is not working
![image](https://github.com/microsoft/vscode-copilot-release/assets/61544853/442f5607-6725-4381-b490-568408b451fa)

The chat works but does not seem to be as efficient as usual

**Comment by james0r** [comment]
> Same for me, autocompelete is not working ![image](https://private-user-images.githubusercontent.com/61544853/304019323-442f5607-6725-4381-b490-568408b451fa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc3MzQ2NTQsIm5iZiI6MTcwNzczNDM1NCwicGF0aCI6Ii82MTU0NDg1My8zMDQwMTkzMjMtNDQyZjU2MDctNjcyNS00MzgxLWI0OTAtNTY4NDA4YjQ1MWZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjEyVDEwMzkxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA0ZWZkMGQ5NjMyNGQ1MTMxMzJjODBmYTU1OWY0MTZlNmM0NmQxNmQ0N2E4YzUzNmE1OGI1Y2M3MjRlYzJjMmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._iYheDaC4RhRc79BICGO06IkD9Pcjao8u989iggUxRc)
> 
> The chat works but does not seem to be as efficient as usual

Yeah I noticed the change in efficiency the other day which may or may not have been after an update. Copilot started just giving me answers to chat within the dialog instead of rendering them in the editor. 

I had to explicitly tell it to write the code to the editor and then it worked.

**Comment by joyceerhl** [comment]
There was an incident for Copilot Completions that resulted in timeouts and should now be resolved. This repository tracks issues for Copilot Chat only. If you're still encountering an issue with completions not working, please report it at https://github.com/community/community/discussions/categories/copilot.
